pandas/tests/resample/test_resampler_grouper.py

```diff
@@ -275,3 +275,23 @@ def test_median_duplicate_columns():
     result = df.resample("5s").median()
     expected.columns = result.columns
     tm.assert_frame_equal(result, expected)
+
+
+def test_same_grouper_on_different_frames():
+
+    df1 = pd.DataFrame(
+        [
+            ["a", 1, 2, "05/29/2019"],
+            ["a", 4, 5, "05/28/2019"],
+            ["b", 2, 3, "05/27/2019"],
+        ],
+        columns=["type", "num1", "num2", "date"],
+    ).assign(date=lambda df: pd.to_datetime(df["date"]))
```

Comment:
are the `assign` calls needed here?  if we aren't trying to test assign, itd be better to just write out df1 and df2 directly

Reply:
no they're not needed. changes `df` construction to keep it simpler.